src/components/common/buttons/SolidButton/style.module.scss

```
@@ -0,0 +1,21 @@
+@import './src/resources/styles/colors';
+@import '../../../../resources/styles/typography';
+
+.solidButton {
+  height: 60px;
+  background: $color-surface-light 0 0 no-repeat padding-box;
+  border-radius: 163px;
+  font: Semibold 18px/23px $typographic-font-family;
+  color: $typography-on-surface-light;
+  box-shadow: 0 2px 6px $color-shadow;
+  transition: box-shadow .3s;
+  text-align: center;
+  text-transform: uppercase;
+
+  &:hover,
+  &:focus {
```

Why focused button would equal to hovered? It seems the button is kind of selected after you have touched (don't go back automaticly to normal state).

```suggestion
  &:hover {
```

---

You are right. [Fixed](https://github.com/lutick/website/pull/10/commits/442faaea79cbf57fe1595f1922fed694e19b89a2).